### DISCUSSION TITLE
Is there a link to the latest image (only)?

### DISCUSSION THREAD

**Discussion by tjzepf** [aaronwmorris/indi-allsky#1731] [Q&A] [answered]
I'm looking to link to the image (not the whole website) in my weather website. Is there a way to get a URL to the "latest" image instead of individually names ones?

**Comment by aaronwmorris** [comment, marked as answer]
There are redirect views that will always issue a 302 redirect to the latest image (or startrail, keogram, etc)

For example:  `https://hostname.local/indi-allsky/latestimage`

https://github.com/aaronwmorris/indi-allsky/wiki/Redirect-Views#latest-image

**Reply by tjzepf** [reply]
Perfect, thank you!
